ios/Podfile.lock

```diff
@@ -286,6 +286,8 @@ PODS:
     - React
   - react-native-background-timer (2.1.1):
     - React
```

Comment:
Don't forget, you don't need to add pod of new libraries but need to run `pod install` on iOS directory, thanks.

Reply:
@djorkaeffalexandre Ok Sir, thanks